core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -355,6 +362,29 @@ public void removeBlock(long sessionId, long blockId)
     mBlockStore.removeBlock(sessionId, blockId);
   }
 
+  public void freeWorker() throws IOException {
+    // TODO(Tony Sun): Here create new tiers, need to be fixed later.
+    List<StorageTier> curTiers = IntStream.range(0, WORKER_STORAGE_TIER_ASSOC.size()).mapToObj(
```

Comment:
Configuration.global().getList()(PropertyKey, TEMPLATE.WORKER_TIERED_STORE_LEVEL_DIRS_PATH.format(i))
and PagedBlockStore: WORKER_PAGE_STORE_DIRS.